pandas/tests/computation/test_eval.py

```diff
@@ -353,16 +353,6 @@ def test_pow(self, lhs, rhs, engine, parser):
             expected = _eval_single_bin(middle, "**", rhs, engine)
             tm.assert_almost_equal(result, expected)
 
-    def check_single_invert_op(self, lhs, engine, parser):
```

Comment:
looks like this was a helped function which wasn't being used anymore, so I've removed it